### PR TITLE
fix: output toggle button

### PR DESCRIPTION
**Pull request by lecepin**
- ~~增加嵌入风格，isEmbeddedMode。~~
- mobile 模式下的， Output/Code 的定制透出。
- 修复响应式下的切换按钮不工作问题

### CODE REVIEW

**Comment by lecepin** [comment]
image

image


嵌入文档等场景使用。

**Comment by lecepin** [comment]
@sxzz 你好，可以帮忙 CR 下么

**Review by sxzz — CHANGES_REQUESTED** [review]
The embed mode is hard to generalize and can be easily achieved with style overrides, so it can be removed.

File: src/Repl.vue [hunk 1]
```diff
@@ -40,6 +40,11 @@ export interface Props {
     showErrorText?: string
     monacoOptions?: monaco.editor.IStandaloneEditorConstructionOptions
   }
+  splitPaneOptions?: {
+    CodeTogglerButtonText?: string
+    OutputTogglerButtonText?: string
+  }
+  isEmbedMode?: boolean
```
Comment (sxzz) [inline src/Repl.vue:47]: As https://github.com/vuejs/repl/pull/279#pullrequestreview-2278144262 said, embed mode can be implemented without any javascript.

File: src/Repl.vue [hunk 2]
```diff
@@ -57,6 +62,11 @@ const props = withDefaults(defineProps<Props>(), {
   layout: 'horizontal',
   previewOptions: () => ({}),
   editorOptions: () => ({}),
+  splitPaneOptions: () => ({
+    CodeTogglerButtonText: '< Code',
+    OutputTogglerButtonText: 'Output >',
+  }),
+  isEmbedMode: false
```
Comment (sxzz) [inline src/Repl.vue:69]: Remove it

File: src/SplitPane.vue [hunk 3]
```diff
@@ -215,4 +220,6 @@ function changeViewSize() {
     pointer-events: none;
   }
 }
+
+
```
Comment (sxzz) [inline src/SplitPane.vue:224]: Remove blanks

**Review by sxzz — CHANGES_REQUESTED** [review]

**Review by sxzz — APPROVED** [review]

**Pull request merged**